alerts/openshift-virtualization-operator/KubeVirtComponentExceedsRequestedCPU.md

```diff
@@ -1,39 +1,4 @@
-# KubeVirtComponentExceedsRequestedCPU
+# KubeVirtComponentExceedsRequestedCPU [Deprecated]
 
-## Meaning
-
-This alert fires when a component's CPU usage exceeds the requested limit.
-
-## Impact
-
-Usage of CPU resources is not optimal and the node might be overloaded.
-
-## Diagnosis
-
-1. Set the `NAMESPACE` environment variable:
-
-   ```bash
-   $ export NAMESPACE="$(oc get kubevirt -A \
-     -o custom-columns="":.metadata.namespace)"
-   ```
-
-2. Check the component's CPU request limit:
-
-   ```bash
-   $ oc -n $NAMESPACE get deployment <component> -o yaml | grep requests: -A 2
-   ```
-
-3. Check the actual CPU usage by using a PromQL query:
-
-   ```text
-   node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate
-   {namespace="$NAMESPACE",container="<component>"}
-   ```
-
-See the
-[Prometheus documentation](https://prometheus.io/docs/prometheus/latest/querying/basics/)
-for more information.
-
-## Mitigation
-
-Update the CPU request limit in the `HCO` custom resource.
\ No newline at end of file
+This alert has been deprecated; it does not indicate an actual issue.
+If triggered, it may be safely ignored and silenced.
```

Comment:
Is this the established text for a deprecated alert? If you're open to changing it, I think we can make it clearer by using minimalism principles.

- I'd suggest using the active voice ("You can safely ignore or silence this alert") instead of the passive voice ("it may be safely ignored and silenced").

- I'd remove "it does not indicate an actual issue" because it adds some confusion, IMO, and "safely" implies that it's a non-issue. 

Suggestion:

```suggestion
This alert is deprecated. You can safely ignore or silence it.
```